Description: Workaround for MYSQLXPLUGIN_DIR in the datadir_permission test
 The test is in the main suite, and includes a custom cnf file that specifies
 --plugin-dir=@env.MYSQLXPLUGIN_DIR, but in 5.7.14 mysqlx  was removed from
 the standard plugin list in mysql-test/include/plugin.defs. So the value is
 not always defined.
Author: Lars Tangvald <lars.tangvald@oracle.com>
Bug: https://bugs.mysql.com/bug.php?id=83321
Last-Update: 2017-04-05

diff --git a/mysql-test/t/datadir_permission.cnf b/mysql-test/t/datadir_permission.cnf
index 18cff76..5c3ea49 100644
--- a/mysql-test/t/datadir_permission.cnf
+++ b/mysql-test/t/datadir_permission.cnf
@@ -2,4 +2,4 @@
 !include include/default_my.cnf
 
 [mysqld.1]
-plugin-dir=@env.MYSQLXPLUGIN_DIR
+#plugin-dir=@env.MYSQLXPLUGIN_DIR
